examples/toolchains/nodejs/shell.nix

```
@@ -1,3 +1,3 @@
 { pkgs ? import ./nixpkgs.nix { } }:
 
-pkgs.mkShellNoCC { nativeBuildInputs = [ pkgs.bazel_6 ]; }
+pkgs.mkShellNoCC { nativeBuildInputs = with pkgs; [ bazel_6 nodejs ]; }
```

```suggestion
pkgs.mkShellNoCC { nativeBuildInputs = with pkgs; [ bazel_6 ]; }
```
Adding `nodejs` to the shell is not necessary for the example to build thanks to the `node_repository` attribute to `npm_install`. It works without in a pure Nix shell.